### ISSUE TITLE
[feature-request] command functionality for tooltips

### ISSUE THREAD

**Issue by revuwa** [vars1ty/HybridBar#17]
Thanks for that cool piece of software ❤️

It would be pretty cool, if HybridBar could show executed commands as tooltips.
I thought on somethin' like: `"tooltip_command": "whoami"`, or whatever.

So one label or button could be used for more than one command.
As examples, so:

- a clock label could tooltip date specific stuff,
- a wifi label could tooltip station connection foo,
- a memory used label could tooltip memory free, or vice versa...

What do you think?

**Comment by vars1ty** [comment]
Sounds easy enough to implement quickly, so could probably look into it this weekend.
Syntax would probably be something like:
```json
{
    "right-label_unnamed_1": {
        "text": "username: ",
        "command": "whomai",
        "tooltip": "Kernel: ",
        "tooltip_command": "uname -r"
    }
}

**Comment by revuwa** [comment]
> Sounds easy enough to implement quickly, so could probably look into it this weekend.

Well, thanks in advance for looking into this.

**Comment by vars1ty** [comment]
Fixed in 8e371636073477a86f9e1ff303a68109d78e59d4 (`0.3.6`) which I'll release shortly 👍🏽

**Comment by revuwa** [comment]
Wow, thank you so much!
